guides/concepts/list-of-operations.md

```diff
@@ -388,9 +400,13 @@ Possible errors:
 ## Manage Data
 [JavaScript](http://stellar.github.io/js-stellar-sdk/Operation.html#.manageData) | [Java](http://stellar.github.io/java-stellar-sdk/org/stellar/sdk/ManageDataOperation.Builder.html) | [Go](https://godoc.org/github.com/stellar/go/txnbuild#ManageData)
 
-Allows you to set,modify or delete a Data Entry (name/value pair) that is attached to a particular account. An account can have an arbitrary amount of DataEntries attached to it. Each DataEntry increases the minimum balance needed to be held by the account.
+Sets, modifies, or deletes a Data Entry (name/value pair) that is attached to a particular account.
+
+An account can have an arbitrary amount of data entries attached to it. Each data entry increases
```

Comment:
"arbitrary" is not strictly true, although the number is large (1000 I think).

Reply:
An account can have a large amount of data entries attached to it (subject to sub-entry limits for
an account). Each data entry increases the minimum balance (via the base reserve) needed to be held
by the account.